src/components/checkbox/Checkbox.tsx

```diff
@@ -17,28 +17,27 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
+
 import styled from '@emotion/styled';
 import * as RadixCheckbox from '@radix-ui/react-checkbox';
 import { useCallback } from 'react';
 import { Spinner } from '../spinner';
 import { CheckboxIcon } from './CheckboxIcon';
 
 interface Props {
+  ariaLabel?: string;
   checked: boolean | 'indeterminate';
   className?: string;
-
-  ariaLabel?: string;
-  label?: string;
+  hasError?: boolean;
   helpText?: string;
-  title?: string;
   id?: string;
-
-  hasError?: boolean;
+  innerClassName?: string;
   isDisabled?: boolean;
   isLoading?: boolean;
-
+  label?: string;
   onCheck: (checked: boolean | 'indeterminate', id?: string) => void;
   onFocus?: VoidFunction;
+  title?: string;
```

Comment:
@jeremy-davis-sonarsource @david-cho-lerat-sonarsource should we enable an eslint rule for that?

Reply:
I wouldn't mind always having the props in alpha order, at least it's a system. I know the opposing arguments of grouping them "semantically", or "functions together", or "important first", etc, but in real life when no order is enforced none of this is followed in the long run and we just end up with "whatever". I'd much prefer to enforce the alpha order and be done with it.

Reply:
Sarath (?) mentioned having the same VScode config for everyone (and every project!), which think makes sense, and will solve this aspect as well

Reply:
I think this is enforced by an eslint rule on SC, not sure the vscode config will help there.
But I guess we also want a common eslint config for all projects.